CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://x.com/revancedapp">
+      <picture>
+         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
+         <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
+      </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://www.youtube.com/@ReVanced">
+       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
+   </a>
+   <br>
+   <br>
+   Continuing the legacy of Vanced
+</p>
+
+# 👋 Contribution guidelines
+This document describes how to contribute to ReVanced Manager.
+
+## 📖 Resources to help you get started
+* The [documentation](/docs/README.md) provides steps to build ReVanced Manager from source
+* Our [backlog](https://github.com/orgs/ReVanced/projects/12) is where we keep track of what we're working on
+* [Issues](https://github.com/ReVanced/revanced-manager/issues) are where we keep track of bugs and feature requests
+
+## 🙏 Submitting a feature request
+Features can be requested by opening an issue using the
+[feature request issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=feature-request&projects=&template=feature-issue.yml&title=feat%3A+%3Ctitle%3E).
+
+> [!NOTE]
+> We may reject your request at the discretion of ReVanced Manager's maintainers,
+> please provide good motivation for a request to be accepted.
+
+## 🐞 Submitting a bug report
+If you encounter a bug while using the ReVanced Manager app, open an issue using the
+[bug report issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=bug&projects=&template=bug-issue.yml&title=bug%3A+%3Ctitle%3E).
+
+## 📝 How to contribute
+> [!NOTE]
+> It's worth noting that we recommend that you discuss your changes with 
+> the maintainers of ReVanced Manager before contributing. 
+> This will help you determine whether your change is acceptable.
+
+1. Fork the repository and create your branch from `dev`
+2. Make sure that the commit name are clear and concise as possible and commit your changes.
+   If you have never contributed to this repository before, it's recommended to read the overview in the [documentation for developers](/docs/developers/README.md)
+3. Open a pull request to the `dev` branch and reference issues that your pull request closes
+4. The maintainers of ReVanced Manager will review and provide suggestions. 
+   Once your pull request is approved and merged, it will be included in the next release of ReVanced Manager
+
+## 🤚 I want to contribute but don't know how to code
+Even if you don't know how to code, you can still contribute by 
+translating the ReVanced Manager application on our [Crowdin](https://translate.revanced.app/).
```

Comment:
`the ReVanced Manager application` can just be `ReVanced Manager`

Reply:
```suggestion
translating ReVanced Manager on [Crowdin](https://translate.revanced.app/).
```


Co-authored-by: Palm <palmpasuthorn@gmail.com>